![SiteBannern](../researcharc.png)

## Research

There are over 1.5 million known species on the planet and likely many times that number of yet undiscovered species. What are the rules of life that gave rise to the diversity we observe? Why do some lineages (e.g., insects) have so many species while others have relatively few? We aim to understand the patterns of species diversity and investigate the processes and mechanisms that give rise to that diversity. We approach this very broad question by integrating several disciplines including genomics, bioinformatics, phylogenetics, and taxonomy; through work in the field and the lab. Our research forms a bridge between the biodiversity community and the comparative genomics community. Our long-term interests include investigating links between ecological factors (e.g., environmental stress), rapid evolution of repetitive elements, and the process of speciation. We use diverse insect groups (especially beetles) as models with which to ask our questions. Alongside our collaborators we investigate repetitive DNA dynamics in a variety of plant animal groups. 

### Repetitive DNA

Large fractions of the genome in most organisms are made up of repetitive DNA elements. These repeated sequences can form blocks which can evolve more quickly than unique DNA sequences (e.g., genes). Repeitive elements have been understudied for decades due to technical and computational challenges associated with their sequencing and assembly. As recent advances in sequencing technology overcome those obstacles, work in model organisms is shedding new light on the critical role of repeats in genome evolution at various scales. However, we know little about how repeat dynamics shape genome evolution in naturally evolving species across changing ecological landscapes. Our research program integrates field work, data generation, and development of new computational tools to investigate comparative repeat genomics across phylogenetic scales in insects, with an emphasis on ground beetles. 

A current focus in the lab is to investigate rapid genome evolution that is triggered by an apparent interaction between ribosomal DNA, transposable elements, and satellite DNA. This work builds on previous findings (https://doi.org/10.1093/sysbio/syaa030) that showed ground beetles in the _breve_ species group of _Bembidion_ are an ideal model for studying this phenomenom that is commonly observed across plants, animals, and fungi.

#### Software develoment

Repetitive DNA genomics is an emerging field, and as such, there are many opportunities to develop new bioinformatic tools. Our software development efforts focus on tools that enable efficient study of repetitive elements across phylogenetic scales. We continue to develop RepeatProfiler (https://github.com/johnssproul/RepeatProfiler) along with colleagues in the Larracuente Lab at the University of Rochester (kudos to the brilliant Sherif Negm, the first author and U of R undergraduate!). Our ongoing work includes development of a pipeline for detecting homology among satDNA sequences within and across species.

![SiteBannern](../RepeatProfiler.png)

New software connections to scale repetitive DNA genomics to diverse clades.

![SiteBannern](../RepeatPipelinesWorkflow.png)

### Biodiversity, species delimitation and taxonomy
Discovering and documenting the planet's millions of species remains one of the grand challenges of science. A deep understanding of what species are present and evolutionary relationships among them lies at the foundation of most fields in biology. We seek to discover the patterns of biodiversity both as a field unto itself (i.e., taxonomy), and as a starting point for studies in repetitive DNA dynamics that rely on the framework of species boundaries established by our taxonomic work. 

Studying repeats in a comparative evolutionary framework that spans populations, species, and diverse clades is a powerful lens for understanding patterns, processes, and mechanisms that underlie genome evolution, repeat biology, and the diversification of life. 
